### PR TITLE
Add client

### PR DESCRIPTION
**Pull request by eliasson**
Initial code for adding client. It is not ready to be merged yet, only for early comments.

Done:
- Rough modal implemented (not fully centered)
- Enabled DRF for Client
- UI for listing and adding

Still left to do:
- View / Edit
- Turn off pagination server side or add pagination to lists
- Search?

### CODE REVIEW

File: quarterapp/ui/static/scripts/quarterapp.min.js [hunk 1]
```diff
@@ -11566,8 +11588,66 @@ function(a, b, c) {
...
+        var service = {
+            clients: clients,
+            createClient: createClient
+        };
+        return service;
+    }
+    angular.module("app.manage").factory("api", apiService), apiService.$inject = [ "$http", "$q" ];
+}(), function() {
+    "use strict";
+    function ClientController($timeout, $state, api) {
```
Comment (dbackman) [inline quarterapp/ui/static/scripts/quarterapp.min.js]: Just a question for the ClientController. why using loading = !0 and !1 instead of false/true?

**Comment by dbackman** [comment]
Good job! I need to catch up some angular detailed knowledge but the structure is easy to follow.

**Comment by eliasson** [comment]
Implemented simple version of edit client. Ok to merge @dbackman?

**Comment by dbackman** [comment]
Yes
Den 12 feb 2016 18:20 skrev "Markus Eliasson" <notifications@github.com>:

**Pull request merged**